src/java.desktop/share/classes/javax/swing/JMenu.java

```diff
@@ -1256,7 +1256,11 @@ public void applyComponentOrientation(ComponentOrientation o) {
     }
 
     /**
-     * {@inheritDoc}
+     * Sets the orientation for this menu and the associated popup component
+     * determined by the <code>ComponentOrientation</code> argument.
+     *
+     * @param o the new component orientation of this menu and
+     *        the popup menu component contained within it.
      */
     public void setComponentOrientation(ComponentOrientation o) {
```

Comment:
For consistency, I suggest using the same wording here as in the introduction sentence.